src/app/migrations/versions/3cfb8cf58e98_update_session.py

```diff
@@ -0,0 +1,41 @@
+"""Update session
+
+Revision ID: 3cfb8cf58e98
+Revises: 110f14cc6f39
+Create Date: 2021-04-04 01:26:03.892507
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '3cfb8cf58e98'
+down_revision = '110f14cc6f39'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('sessions',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('device_token', sa.String(), nullable=True),
+    sa.Column('device_type', sa.String(), nullable=True),
+    sa.Column('session_token', sa.String(), nullable=False),
+    sa.Column('session_expiration', sa.DateTime(), nullable=False),
+    sa.Column('update_token', sa.String(), nullable=False),
+    sa.Column('last_used', sa.DateTime(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=True),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('session_token'),
+    sa.UniqueConstraint('update_token')
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
whaaa doc string

Reply:
i think these are literally autogenerated by alembic, including the comments